gateways/firstdata/translation.go

```diff
@@ -0,0 +1,40 @@
+package firstdata
+
+import "github.com/BoltApp/sleet"
+
+// translateCvv converts a Firstdata CVV response code to its equivalent Sleet standard code.
+// TODO do case for NOT_CERTIFIED
+func translateCvv(rawCvv string) sleet.CVVResponse {
+	switch rawCvv {
+	case "MATCHED":
```

Comment:
what if we simply had a map we could get value from map and if not present return unknown? easier to read

```
var firstdataCvvMap = map[string]sleet.CVVResponse{
  "MATCHED": sleet.CVVResponseMatch,
  "NOT_MATCHED": sleet.CVVResponseNoMatch,
  ...
}

cvvCode, ok := firstdataCvvMap[rawCvv]
if !ok {
	return sleet.CVVResponseUnknown
}
return cvvCode
```

Reply:
Maps are great, I should actually update other gateways to do this...